ui/src/settings/components/playground_flyout/PlaygroundFlyout.scss

```diff
@@ -1,35 +1,6 @@
-@import "~@elastic/eui/src/global_styling/variables/animations";
-@import "~@elastic/eui/src/global_styling/variables/colors";
-@use "~@elastic/eui/src/global_styling/utility/animations";
-
-.playgroundFlyoutFooter {
-  box-shadow: inset 5px 0 9px -9px rgba(0, 0, 0, 0.4);
-}
-
-.playgroundFlyout--right {
-  animation-name: euiFlyoutRight;
-  animation-duration: 0s;
-  border-left: 0;
-  border-right: 0;
-  box-shadow: inset 5px 0 9px -9px rgba(0, 0, 0, 0.4);
-  scrollbar-width: thin;
-  position: sticky;
-  min-height: calc(100vh - 49px);
-  max-height: calc(100vh - 49px);
-  min-width: 313px !important;
-  max-width: 313px !important;
-  top: 49px; // EuiNavBar is 49px
-  z-index: 1;
-}
-
-@keyframes euiFlyoutRight {
-  0% {
-    opacity: 0;
-    transform: translateX(+100%);
-  }
-
-  75% {
-    opacity: 1;
-    transform: translateX(0%);
-  }
+// Required for setting a specific flyout width that is smaller than the default EUI flyout width (medium); overwrites
+// the min-width that corresponds to the default EUI flyout width
+.euiFlyout--medium {
+  width: 313px;
+  min-width: unset;
```

Comment:
Moved the custom flyout widths here.

Reply:
Same comment as this: https://github.com/caraml-dev/xp/pull/33/files#r967945038

Reply:
Fixed this here https://github.com/caraml-dev/xp/blob/0a6fa979d645a743abc4c82f9293f6242a36ce52/ui/src/settings/components/playground_flyout/PlaygroundFlyout.scss